ani-cli

```diff
@@ -44,32 +49,47 @@ help_text () {
 }
 
 version_text () {
-    printf "$c_blue%s$c_reset%s\n" "Version: " "$VERSION"  >&2
+    inf "Version:" "$VERSION"  >&2
 }
 
-
 die () {
-	printf "$c_red%s$c_reset\n" "$*" >&2
+	err "$*"
 	exit 1
 }
 
-err () {
-	printf "$c_red%s$c_reset\n" "$*" >&2
-}
-
 update_script () {
 	# get the newest version of this script from github and replace it
 	update="$(curl -s "https://raw.githubusercontent.com/pystardust/ani-cli/master/ani-cli" | diff -u "$0" -)"
 	if [ -z "$update" ]; then
-		printf "$c_green%s$c_reset\n" "Script is up to date :)"
+		inf "Script is up to date :)"
 	else
-		printf '%s\n' "$update" | patch "$0" - && printf "$c_green%s$c_reset\n" "Script has been updated" ||
-            printf "$c_red%s$c_reset\n" "Can't update for some reason!"
+		printf '%s\n' "$update" | patch "$0" - && inf "Script has been updated" ||
+            err "Can't update for some reason!"
 	fi
 }
 
+dep_ch () {
+	# checks if programs are present 
+	for dep; do
+		if ! command -v "$dep" >/dev/null ; then
+			err "Program \"$dep\" not found. Please install it."
+			#aria2c is in the package aria2
+			if [ "$dep" = "aria2c" ]; then
+				# echo, inf or err?
+				echo "To install aria2c, Type <your_package_manager> aria2"
```

Comment:
Should be err / die. die makes most sense since that gets rid of the `exit 1` on line 81

Reply:
The `exit 1` is outside of the `if`-`fi` branch, so using die wouldn't replace it. Using die on line 75 is a bad idea too, since it'd cause the following lines to be never executed.
On the other hand instead of `exit 1` an empty `die` would make sense.